src/studio/src/load-balancer/Dockerfile

```diff
@@ -1,4 +1,4 @@
 # this dockerfile is only used for local development
-FROM nginx:1.19.7-alpine
+FROM nginx:1.19.8-alpine
```

Comment:
This value should be updated in our devops repo too: https://dev.azure.com/brreg/_git/altinn-studio-ops?path=%2Fdeploy%2Faltinn-studio%2Faltinn-loadbalancer%2Fvalues.yaml

Reply:
updated :)